### ISSUE TITLE
Add KFG to Linkedin and set to Linnea's employer <3

### ISSUE THREAD

**Issue by seekanddefine** [knowledge-for-good/knowledgeforgood#56]
*   [x] #255
*   [x] #256

Linkedin is an amazing place to connect to other educators around the world and people invested in edtech work. More than any other platform I see connections happening here so I think it is an excellent first place after Github for KFG to start moving and sharing. Since our main product right now is workshops, it will also potentially benefit workshop attendance by connecting with more people and keeping that connection consistent.

**Comment by seekanddefine** [comment]
Opening again. We can use github as our website for now and we have the [knowledgeforgood@gmail.com](mailto:knowledgeforgood@gmail.com) email we can use for the time being, but now there is a different block. I went to sign up and it says that the name is already taken but there is no company with the name found on linkedin. I'm wondering if I already started claiming it and now need to finish the registration but am not sure how to go back. OR maybe @avidrucker started the process?
